templates/search/results-contextual-en.html

```diff
@@ -29,11 +29,10 @@
 			</div>
 			<div class="clearfix"></div>
 		</div>
+		<p id="pers-info" class="mrgn-tp-md">Please <strong>do not include sensitive personal information</strong> in the search box, such as your social insurance number, personal finance data and medical or work history.</p>
```

Comment:
The `id` will need to modified to include more context. This is to avoid collision and quickly know it's context of use.

Reply:
I recommend `#gc-pi`

CC: @Ricokola @duboisp